linera-service/tests/end_to_end_tests.rs

```diff
@@ -938,21 +943,32 @@ impl NodeService {
         value["data"].clone()
     }
 
-    async fn create_application(&self, bytecode_id: &str) -> String {
+    async fn create_application(
+        &self,
+        bytecode_id: String,
+        parameters: String,
+        argument: String,
+        required_application_ids: Vec<String>,
+    ) -> String {
+        let json_required_applications_ids =
+            serde_json::to_string(&required_application_ids).unwrap();
+        let new_parameters = parameters.replace('\"', "\\\"");
```

Comment:
This kind of replacement (string escaping) is generally tricky to get right. Maybe that's fine for this PR but we should create a task to stop using strings.

Reply:
Ok, not sure how to avoid, but I created the issue.